### DISCUSSION TITLE
S3 Generic Storage with OVH Object Storage Error

### DISCUSSION THREAD

**Discussion by Starmism** [requarks/wiki#5830] [Error / Bug Report]
![Error in console](https://i.imgur.com/jKZb1u1.png)

GraphQL error which is bizarre, everything is filled out on the page using credentials I know work as I use them in my Django website. Anything else I can do to help diagnose this? I would very much like to get away from Local File Storage to the S3 provider I pay for. Thanks!

Update: Got the dev setup, trying it out, not getting much. Though I did get an error in the UI: ![ui error](https://i.imgur.com/wpKIUy4.png)

**Comment by Spivoran** [comment]
I am getting the same error using Digital Ocean Spaces. Only the "Path" line is different in the dev console: `['storage', 'status', '0', 'message']`. When I deactivate the storage module, the error goes away.

Just jumping in to monitor the conversation. At the end of the day, I would just like to back up my pages and static files to deploy them to an installation of another instance of WIKI.JS on a different server. Haven't had much luck setting up Local Storage either.
